Makefile

```diff
@@ -46,9 +46,9 @@ tetragon-bpf-verify: tetragon-bpf
 	sudo contrib/vmtest/tetragon-verify-programs bpf/objs
 
 tetragon-bpf-container:
-	docker rm hubble-llvm || true
-	docker run -v $(CURDIR):/tetragon -u $$(id -u)  --name hubble-llvm $(CLANG_IMAGE) $(MAKE) -C /tetragon/bpf
-	docker rm hubble-llvm
+	$(CONTAINER_ENGINE) rm hubble-llvm || true
+	$(CONTAINER_ENGINE) run -v $(CURDIR):/tetragon -u $$(id -u)  --name hubble-llvm $(CLANG_IMAGE) $(MAKE) -C /tetragon/bpf
+	$(CONTAINER_ENGINE) rm hubble-llvm
```

Comment:
Is CONTAINER_ENGINE always set? I don't see it in my environment. Maybe we should have a default value (docker probaby) in the `Makefile`  for it?

Also, what other container engines could be used in this case? Would they work with the current images?

Reply:
>Maybe we should have a default value (docker probaby) in the Makefile for it?

That's exactly what happens already. `CONTAINER_ENGINE` set to the default value `docker` in https://github.com/cilium/tetragon/blob/main/Makefile#L4 if not provided as a environment variable or command-line argument for make, for example `make CONTAINER_ENGINE="sudo podman"`.  The `sudo podman` setting does work - tested on RHEL8.  There is a known remaining issue with `Makefile.docker` but it can be addressed separately (there's a warning `docker not found` but it doesn't preclude the successful compilation).

Reply:
ah, got it. Thanks!